frame.go

```diff
@@ -55,6 +61,63 @@ func (f *Frame) Data() [NumDataPointers][]byte {
 	return b
 }
 
+// DataBytes returns the frame's data as byte slice.
+func (f *Frame) DataBytes() ([]byte, error) {
+
+	isVideoFrame := f.Height() > 0 && f.Width() > 0
+
+	if isVideoFrame {
+
+		// retrieve the image buffer size
+		bufferSize := C.av_image_get_buffer_size(C.enum_AVPixelFormat(f.c.format), f.c.width, f.c.height, 1)
+		if bufferSize < 0 {
+			return nil, newError(bufferSize)
```

Comment:
Could you wrap the error here ?

`fmt.Errorf("astiav: getting image buffer size failed: %w", newError(bufferSize))`